### ISSUE TITLE
[a11y] - toaster Insufficient contrast foreground and background

### ISSUE THREAD

**Issue by transkamp** [open-pioneer/trails-openlayers-base-packages#450]
### Issue description

open https://open-pioneer.github.io/trails-demo/core-packages/dev/samples/notify-sample/

the foreground and background colors of the toaster sometimes do not have the necessary minimum contrast.
To be clear. at least level `AA `must be reached. under the link here you can find a comparison in grid form with alternative colors to pass the requirement - [contrast grid](https://contrast-grid.eightshapes.com/?version=1.1.0&background-colors=%23dc2626%2C%20error%0D%0A%23AB1B1B%2C%20error%20alternativ%201%0D%0A%23ea580c%2C%20warning%0D%0A%23FF6D1F%2C%20warning%20alternativ%201%0D%0A%2316A34A%2C%20success%0D%0A%2379DF9E%2C%20sucess%20alternativ%201%0D%0A%230C642C%2C%20sucess%20alternativ%202%0D%0A%23fff%2C%20info%0D%0A%0D%0A%0D%0A%0D%0A&foreground-colors=%23000%2C%20text%20black%0D%0A%23fff%2C%20text%20white&es-color-form__tile-size=regular&es-color-form__show-contrast=aaa&es-color-form__show-contrast=aa&es-color-form__show-contrast=aa18&es-color-form__show-contrast=dnp)

![Image](https://github.com/user-attachments/assets/a4722703-4d6b-457a-9735-e34390c7055f)

### Validations

- [x] Read and follow our [code of conduct](https://github.com/open-pioneer/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [contributing guidelines](https://github.com/open-pioneer/trails-starter/blob/main/docs/Contributing.md)
- [x] Check that there isn't already a similar issue
